src/commandline_content.ts

```diff
@@ -34,21 +35,17 @@ document.addEventListener("DOMContentLoaded", init)
 // This second call will fail in the most common case, but makes web-ext reloads effective.
 init()
 
-// The CSS in this file should probably go in static/
 export function show(){
-    // I don't understand how CSS works - but this ensures that the commandline is always at the bottom of the page.
-    cmdline_iframe.setAttribute("style", "position: fixed; bottom: 0; left: 0; z-index: 10000; width: 100%; height: 24px; border: 0; padding: 0; margin: 0;");
```

Comment:
I just remove this and don't find any issue, so how can this issue recur? :)

Reply:
I can't reproduce the issue either on current master.

It is still reproducible at 242fcce, though, where commenting out that line will stop the command line from showing. Something somewhere else must have fixed the underlying issue.